webpage/src/installation/fedora.md

```diff
@@ -53,3 +53,16 @@ dnf install qownnotes
 ```
 
 [Direct Download](https://download.opensuse.org/repositories/home:/pbek:/QOwnNotes/Fedora_40) (this example link is for Fedora 40)
+
+## QON Version Updating Notes for Fedora
```

Comment:
Can you please write `QOwnNotes` instead of QON. 😉

Reply:
> Can you please write `QOwnNotes` instead of QON.

It looks like you have already done that, thanks.

I am attempting to add some minor clarifying information, fix some grammar and also add/edit some of the original punctuation which is not quite "perfect" English.

Unfortunately, I am having problems editing my fork now that I synced it with yours. :-/